releasenotes/notes/add-tls-enhanced-tags-6ff09ae7fc0ff7a1.yaml

```
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    The agent will now tag TLS enhanced metrics like `tls_version` and `tls_cipher`.
+    This will allow you to filter and aggregate metrics based on the TLS version and cipher used in the connection.
+    The tags will be added in NPM and USM.
```

```suggestion
    The tags will be added in CNM and USM.
```
NPM is now CNM (Cloud Network Monitoring)

---

😮